contracts/adapters/FlexibleLeverageStrategyAdapter.sol

```diff
@@ -645,16 +656,16 @@ contract FlexibleLeverageStrategyAdapter is BaseAdapter {
             newLeverageRatio = _calculateNewLeverageRatio(currentLeverageRatio);
         }
 
-
         ActionInfo memory actionInfo = _createActionInfo();
 
         sizes = new uint256[](_exchangeNames.length);
-        isLevers = new bool[](_exchangeNames.length);
-        
+
         for (uint256 i = 0; i < _exchangeNames.length; i++) {
 
-            ExchangeSettings memory settings = exchangeSettings[_exchangeNames[i]];
-            uint256 maxTradeSize = isRipcord ? settings.incentivizedTwapMaxTradeSize : settings.twapMaxTradeSize;
+            uint256 maxTradeSize = isRipcord ? 
+                exchangeSettings[_exchangeNames[i]].incentivizedTwapMaxTradeSize : 
+                exchangeSettings[_exchangeNames[i]].twapMaxTradeSize;
+                
             uint256 slippageTolerance = isRipcord ? incentive.incentivizedSlippageTolerance : execution.slippageTolerance;
```

Comment:
Since this isn't used anywhere else let's just move this logic into the `LeverageInfo` declaration

Reply:
I guess same goes for `maxTradeSize`